### ISSUE TITLE
Cross-validation metrics

### ISSUE THREAD

**Issue by TNanukem** [oegedijk/explainerdashboard#97]
First of all, congratulations on this awesome tool!

It came to my attention that the explainer classes do not allow for a cv parameter (either an int or a sklearn class) to be inserted. My request for this feature is related to the metrics that are calculated.

If I do not have an explicitly hold-out test set, the explainer will make the predictions on the train set, therefore, the metrics (R², accuracy, etc) will be overly optimistic. When using cross-validation, the model could be fitted on the k-1 folds and then used to predict the hold-out fold, this way we could have a test estimate using only the training set. The same goes for the residuals plot and others.

Does this implementation make sense for this project? 

Thanks!

**Comment by oegedijk** [comment]
Hi @TNanukem,

Currently there is actually already that functionality for calculating cross validated permutation importances (by passing the `permutation_cv` parameter, but I could also implement this for the various metrics indeed.

Will put it on the roadmap!

**Comment by oegedijk** [comment]
Hi @TNanukem, I think I got it working in the dev branch, so should be in the next release!
